static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -194,51 +303,92 @@ class Issues extends Component<Props, State> {
-                data-test-id={`filter-${value}`}
-                eventKey={value}
-                isActive={value === issuesType}
+                {t('New Issues')}
+                <QueryCount count={count.firstRelease} max={99} hideParens hideIfEmpty />
+              </Button>
+              <Button
+                barId={IssuesType.RESOLVED}
+                size="small"
+                onClick={() => this.handleIssuesTypeSelection(IssuesType.RESOLVED)}
+              >
+                {t('Resolved Issues')}
+                <QueryCount count={count.resolved} max={99} hideParens hideIfEmpty />
+              </Button>
+              <Button
+                barId={IssuesType.UNHANDLED}
+                size="small"
+                onClick={() => this.handleIssuesTypeSelection(IssuesType.UNHANDLED)}
+              >
+                {t('Unhandled Issues')}
+                <QueryCount count={count.unhandled} max={99} hideParens hideIfEmpty />
+              </Button>
+              <Button
+                barId={IssuesType.ALL}
+                size="small"
+                onClick={() => this.handleIssuesTypeSelection(IssuesType.ALL)}
               >
-                {label}
-              </StyledDropdownItem>
-            ))}
-          </DropdownControl>
+                {t('All Issues')}
+                <QueryCount count={count.release} max={99} hideParens hideIfEmpty />
+              </Button>
+            </StyledButtonBar>
+          ) : (
+            <DropdownControl
+              button={({isOpen, getActorProps}) => (
+                <StyledDropdownButton
+                  {...getActorProps()}
+                  isOpen={isOpen}
+                  prefix={t('Filter')}
+                  size="small"
+                >
+                  {issuesTypes.find(i => i.value === issuesType)?.label}
+                </StyledDropdownButton>
+              )}
+            >
+              {issuesTypes.map(({value, label}) => (
+                <StyledDropdownItem
+                  key={value}
+                  onSelect={this.handleIssuesTypeSelection}
+                  data-test-id={`filter-${value}`}
+                  eventKey={value}
+                  isActive={value === issuesType}
+                >
+                  {label}
+                </StyledDropdownItem>
+              ))}
+            </DropdownControl>
+          )}
 
           <OpenInButtonBar gap={1}>
             <Button to={this.getIssuesUrl()} size="small" data-test-id="issues-button">
               {t('Open in Issues')}
             </Button>
 
             <GuideAnchor target="release_issues_open_in_discover">
-              <DiscoverButton
-                to={this.getDiscoverUrl()}
-                size="small"
-                data-test-id="discover-button"
-              >
-                {t('Open in Discover')}
-              </DiscoverButton>
+              {hasReleaseComparison ? null : (
```

Comment:
should we move `GuideAnchor` to be inside this condition so we won't render an empty element?